### ISSUE TITLE
Improve build instructions for CentOS 7 users?

### ISSUE THREAD

**Issue by bradcray** [chapel-lang/chapel#19588]
An Arkouda CentOS7 user was having challenges building Chapel due to the default g++ not supporting the `-std=c++14` flag (or, more to the point, not supporting modern C++).  That led to the following exchange on the current text in the prereqs.rst file, which includes manual workarounds that can be used: https://github.com/mppf/chapel/commit/d1cedf8bbca8cc04d3fc4e9a321f4406ee7e31a5#r70090837

This issue asks whether we could/should improve prereqs.txt for such users or do something else, and in the meantime can serve as help for such users who are looking for it.

**Comment by lydia-duncan** [comment]
I believe we don't support CentOS 7 any more and so could close this issue.  @jabraham17 or @riftEmber - is that correct?

**Comment by jabraham17** [comment]
Yes thats correct. The oldest CentOS we list in the `prereqs` is 9, and CentOS 7 is EOL

**Comment by lydia-duncan** [comment]
Cool, thanks!
